config/tech-docs.yml

```diff
@@ -0,0 +1,49 @@
+# Human read-able name for this site, displayed in the header bar
+service_name: CTC Guarantee Balance Phase5 Service Guide
+
+# The route that will be used to access this site (with tailing slash)
+service_link: /guides/ctc-guarantee-balance-phase5-service-guide/
+
+# ================================================
+# You shouldn't need to edit anything below here
+# ================================================
+
+# Host to use for canonical URL generation (without trailing slash)
+host: developer.service.hmrc.gov.uk
+
+# Header-related options
+show_govuk_logo: true
+
```

Comment:
Add the following here:
`phase: Beta`

This will add "BETA" to top-level navigation of document. This is because all phase 5 documentation needs to have Beta status indicated until go-live. I discussed the issue with James yesterday who raised it with Vince. I will have to ensure that all phase 4 documents no longer have this flag and ensure that all phase 5 documents currently in PROD have it.

Reply:
Will add.